### PR TITLE
Update build tools installation instructions 

### PR DESCRIPTION
**Pull request by aegamesi**
Fixes the protobuf installation command for macOS and removes some redundant instructions

### CODE REVIEW

File: docs/_Dev.md [hunk 1]
```diff
@@ -77,12 +77,6 @@ $ brew install protobuf
 On other operating systems you can directly download the protoc binary
```
Comment (kklin) [inline docs/_Dev.md]: Nit - the commit body should be word wrapped at 80 characters. Also, like in the other commits, the first character after the colon should be capitalized (`docs: Remove redundant...`)

I also noticed that your email in the commit is `aegamesi@users.noreply.github.com`, maybe you should use your Quilt email? I don't feel strongly about it if you have some reason to leave it as is

File: docs/_Dev.md [hunk 2]
```diff
@@ -61,10 +61,25 @@ the one above, you're all set.  If it instead returns someplace else, e.g.,
 
 ### Building Quilt
 
-To build quilt, run `go install` in the Quilt directory. To do things beyond
+To build Quilt, run `go install` in the Quilt directory. To do things beyond
 basic build and install, several additional build tools are required.  These
 can be installed with the `make go-get` target.
 
+You must also tell Node.js to use your local development copy of the Quilt
```
Comment (kklin) [inline docs/_Dev.md]: I'd preface this by saying something like "If developing the @quilt/quilt package`

**Comment by aegamesi** [comment]
@kklin Updated (and fixed commits)

File: docs/_Dev.md (hunk 1 above)
Comment (aegamesi) [reply docs/_Dev.md]: Updated (and fixed commits)

**Pull request merged**